shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/mysql/dal/MySQLAnalyzeTableStatement.java

```diff
@@ -17,12 +17,12 @@
 
 package org.apache.shardingsphere.sql.parser.sql.dialect.statement.mysql.dal;
 
-import org.apache.shardingsphere.sql.parser.sql.common.statement.AbstractSQLStatement;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.dal.AnalyzeTableStatement;
 import org.apache.shardingsphere.sql.parser.sql.common.statement.dal.DALStatement;
 import org.apache.shardingsphere.sql.parser.sql.dialect.statement.mysql.MySQLStatement;
 
 /**
- * Analyze table statement.
+ * MySQL analyze table statement.
  */
-public final class MySQLAnalyzeTableStatement extends AbstractSQLStatement implements DALStatement, MySQLStatement {
+public final class MySQLAnalyzeTableStatement extends AnalyzeTableStatement implements DALStatement, MySQLStatement {
```

Comment:
Hi @strongduanmu 
Where is `MySQLAnalyzeTableStatement` used?

Reply:
> Hi @strongduanmu
> Where is `MySQLAnalyzeTableStatement` used?

@tristaZero Now, `MySQLAnalyzeTableStatement` is only used in the `MySQLDALVisitor` class for parsing. The execution of `anlyze table statement` is not yet supported.

Reply:
Thanks for your explanation. Merged.